hcx/hcxcomponents.md

```diff
@@ -0,0 +1,50 @@
+# VMware HCX Components
+
+## HCX Connector & HCX Cloud Manager
+![HCX Connector](.\images\hcx-connector.png)
```

Comment:
Done of the images load.  They are all 404

Reply:
Fixed